uwblueprint/feeding-canadian-kids
# Submit Button Loading Indicator

## Notion Ticket Link

[Ticket URL](https://www.notion.so/uwblueprintexecs/Add-loading-indicator-to-the-create-meal-request-button-83e703863b474eb189121e84633f9c4d?pvs=4)



## Implementation Description
* Disabled the submit button on Create Meal Request page after it's been pressed once
* Added a loading icon to the button to indicate that the request is still processing



## Steps To Test
1. Go to "Create A Meal Request" on the home page
2. Enter any required details for the "Scheduling" and "Meal Donation information" page
3. On the "Review and Submit" page, click on the submit button and ensure that the button can only be pressed once.



## What Should Reviewers Focus On?
* Reviewers should focus on the Submit button on the "Review and Submit" page (would like a second opinion to see if there should be any UI changes)


## Checklist
- [✓] My PR name is descriptive and in imperative tense
- [✓] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [✓] I have run the appropriate linter(s)
- [✓] I have added tests for my changes
- [✓] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
